platforms/karaf/features/src/main/resources/features.xml

```diff
@@ -2416,10 +2416,10 @@
   <feature name='camel-zipkin' version='${project.version}' resolver='(obr)' start-level='50'>
     <details>camel-zipkin does not work in OSGi, see more at CAMEL-12088</details>
     <feature version='${project.version}'>camel-core</feature>
-    <bundle dependency='true'>mvn:io.zipkin.brave/brave/${brave-zipkin-version}</bundle>
     <bundle dependency='true'>mvn:io.zipkin.brave/brave-core/${brave-zipkin-version}</bundle>
     <bundle dependency='true'>mvn:io.zipkin.brave/brave-spancollector-scribe/${brave-zipkin-version}</bundle>
     <bundle dependency='true'>mvn:io.zipkin.java/zipkin/${zipkin-version}</bundle>
```

Comment:
oops I think this is unneeded now. Will remove

Reply:
Yes, I think so

Reply:
ok scrubbed